plugins/Number.py

```diff
@@ -86,6 +86,7 @@ def init(self, logger):
         self.MaxspeedExtraValue = ["none", "default", "signals", "national", "no", "unposted", "walk", "urban", "variable"]
         self.MaxspeedClassValue = re.compile(u'^[A-Z]*:')
         self.MaxheightExtraValue = ["default", "below_default", "no_indications", "no_sign", "none", "unsigned"]
+        self.MaxweightExtraValue = ["default", "none", "no", "unsigned"]
```

Comment:
```suggestion
        self.MaxweightExtraValue = ["unsigned"]
```

I don't mind if `none` also stays behind, but I don't have new arguments asides from the arguments already given, so this is my review until there's a documented extra entry that should be allowed.

Based on the [wiki](https://wiki.openstreetmap.org/wiki/Key%3Amaxweight): 
> In cases of some objects, for example bridges it may be desirable to tag that place was surveyed and no max weight sign was found. Tag used for this include maxweight=[unsigned](https://wiki.openstreetmap.org/w/index.php?title=Tag:maxweight%3Dunsigned&action=edit&redlink=1) and [maxweight:signed](https://wiki.openstreetmap.org/wiki/Key:maxweight:signed)=[no](https://wiki.openstreetmap.org/wiki/Tag:maxweight:signed%3Dno). There is also maxweight=[none](https://wiki.openstreetmap.org/w/index.php?title=Tag:maxweight%3Dnone&action=edit&redlink=1), but it is misleading as there are also default max weight of vehicles set by law, it is unlikely to find place where one is actually allowed to drive vehicles without any weight limits. However, it is the value used with maxweight:[conditional](https://wiki.openstreetmap.org/wiki/Key:conditional)=none @ * so should be treated as meaning defaults apply.

There's no logical reason why the list should be "consistent" with the maxheight rules above. They're different tags, thus with different allowed values.